doc/README.md

```diff
@@ -2,12 +2,12 @@
 
 This documentation is automatically built and uploaded to GitHub Pages.
 
-To see the documentation locally, type:
+To build and see the documentation locally, type:
 
 ```
 cd doc
 make html
-python -m http.server 8880 --directory ../generated/doc/html &
+python -m http.server 8880 -b 127.0.0.1 --directory generated/html &
 ```
 
 Then open http://localhost:8880/ with your browser.
```

Comment:
```suggestion
Then open http://127.0.0.1:8880/ with your browser.
```

If we tell the server to bind to 127.0.0.1, we should also go there I think. (We can also try `-b localhost` instead.)

Reply:
Then we should go for localhost.